pytext/utils/onnx_utils.py

```diff
@@ -32,12 +39,14 @@ def pytorch_to_caffe2(
     torch.onnx.export(
         model,
         export_input,
-        export_path,
+        export_onnx_path or export_caffe2_path,
```

Comment:
now that we already have export_onnx_path, just always export to it to make it clear